database/seed.js

```diff
@@ -13,59 +11,60 @@ const circleColors = ['#6c8ae4', '#d86441', '#bb6acd', '#df4e96'];
 
 const jabber = new Jabber(buzzWords, 2);
 
-const createReview = (resId, revId) => {
+const createReview = (reviewIdTracker) => {
   const review = {};
-  review._id = revId;
-  review.restaurantId = resId;
+
+  // review
+  review.id = reviewIdTracker;
+  review.overall = faker.random.number({min:1, max: 5});
+  review.food = faker.random.number({min:1, max: 5});
+  review.service = faker.random.number({min:1, max: 5});
+  review.ambience = faker.random.number({min:1, max: 5});
+  review.dineDate = moment(faker.date.between('2019-01-01', '2020-01-24')).format('YYYY-MM-DD');
+  review.noise = faker.random.number({min:1, max: 5});
+  review.recommend = Math.random() < 0.7;
+  review.comments = jabber.createParagraph(faker.random.number({min:10, max: 50}));
+  review.filterTag = faker.random.arrayElement(tags);
+
+  // person
+  review.numReviews = faker.random.number({min:1, max:25});
+  review.color = faker.random.arrayElement(circleColors);
```

Comment:
will all this eventually be moved to a generate data file, and invoked in seed.js ?